arches/app/templates/change_password.htm

```diff
@@ -1,64 +1,57 @@
-<!--
-ARCHES - a program developed to inventory and manage immovable cultural heritage.
-Copyright (C) 2013 J. Paul Getty Trust and World Monuments Fund
-
-This program is free software: you can redistribute it and/or modify
-it under the terms of the GNU Affero General Public License as
-published by the Free Software Foundation, either version 3 of the
-License, or (at your option) any later version.
-
-This program is distributed in the hope that it will be useful,
-but WITHOUT ANY WARRANTY; without even the implied warranty of
-MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
-GNU Affero General Public License for more details.
-
-You should have received a copy of the GNU Affero General Public License
-along with this program. If not, see <http://www.gnu.org/licenses/>.
--->
 {% load static %} {% load i18n %} {% block loading_mask %} {% endblock loading_mask %} {% block body %}
 
     <!-- LOGIN FORM -->
 
     <div class="change-password-form panel panel-default">
-    <div class="panel-heading">{% trans "Change your password" %}</div>
-            <div class="panel-body">
-                <form action="?next={{ next }}" method='POST' {% if auth_failed %} autocomplete="off" {% endif %}>
-                    {% csrf_token %}
-                    <div class="form-group">
-
-                        <div>
-                            <input type="password" class="form-control input-lg" placeholder="Old Password" data-bind="textInput:credentials.old_password" name='old_password' autofocus>
+        <div class="panel-heading">{% trans "Change your password" %}</div>
```

Comment:
The password reset form has more empty space that pushes it below the bottom of the page. The form inputs are no longer centered and placeholders are no longer center aligned. In these two screenshots, the shorter one is in dev/7.4.x and the taller is in the accessibility branch.

<img width="253" alt="image" src="https://github.com/archesproject/arches/assets/1877663/f54fad58-1ae9-42f8-886d-b9ea994c3bba">

<img width="253" alt="image" src="https://github.com/archesproject/arches/assets/1877663/2f59c2aa-9651-449d-ae9a-ff01474590b5">

Reply:
@chiatt Yeah, this seems like a regression.

Reply:
This should now be fixed